using_images/other_images/jenkins.adoc

```diff
@@ -727,6 +727,11 @@ specified.
 <9> The node stanza references the name of the Pod template newly defined above.
 ====
 
+By default the pod is deleted when the build completes.
+This behavior can be modified via the plug-in or within a pipeline Jenkinsfile - see 
+xref:../../using_images/other_images/jenkins_slaves.adoc#slave-pod-retention[Slave Pod Retention]
```

Comment:
Are we still referring to these as Jenkins "Slaves" or can it be changed to "Agents"?

Reply:
Looks like "Agents" is clearer - I can change it